### DISCUSSION TITLE
About this project / Similar Efforts

### DISCUSSION THREAD

**Discussion by Ivorforce** [pfusik/cito#174] [General]
Hi! I just got linked this project and I'm impressed! It looks like you've put quite a bit of work and thought into it.

I'm actually working on a loosely similar project ([Monoteny](https://github.com/Ivorforce/Monoteny)). From a cursory look, I think we're both creating a library-only language with many target environments, although our goals and priorities on the languages (naturally) differ.
Would you be open to a chat about the projects? I'd be interested to know more about your plans and thoughts.

**Comment by pfusik** [comment]
Hi!
Thanks for kind words and sharing your project! It looks like you have only a Python target at the moment?

We can chat here. Or, if you want more interaction, we can schedule a chat at your Discord.

**Reply by Ivorforce** [reply]
Correct. Monoteny is still in heavy development, and likely will be for a while to come.

A chat on Discord sounds great! You can join here: https://discord.gg/gYnEYNZe
